frontend/index.html

```diff
@@ -5,8 +5,48 @@
 </head>
 <body>
 	<script type="text/javascript">
-		var	object_id = Date.now();
-		alert(object_id);
+		let url = "https://policydriverstorage.free.beeceptor.com/";
+		function uploadFile(){
+			fetch(url + "new", {
+				method: "POST",
+				headers: {
+					"Content-Type": "application/json"
+				},
+				body: JSON.stringify({
+					content: document.getElementById('myfile').value,
+					policy: "RAID1"
+				})
+			})
+			.then((response) => response.json())
+ 			.then((json) => console.log(json));
+		}
+
+		function downloadFile(){
+			fetch(url + "download?id=" + document.getElementById('downloadobjid').value)
+	 		.then(response => response.json())
+	 		.then(json => console.log(json));
```

Comment:
Since Download API wont return json format response, would you replace lines of code with navigating to download url. For example:
`window.location.href = url + "download?id=" + document.getElementById('downloadobjid').value;`
Then will be redirected to the download link.